satellite/orders/service.go

```diff
@@ -204,7 +205,11 @@ func (service *Service) CreateGetOrderLimits(ctx context.Context, uplink *identi
 		return nil, Error.Wrap(err)
 	}
 
-	if err := service.updateBandwidth(ctx, bucketID, limits...); err != nil {
+	projectID, bucketName, err := SplitBucketID(bucketID)
```

Comment:
The bucket id shouldn't need to be split the argument should already be split.

Reply:
bucketID gets passed in as an arg to this function

Reply:
Ah, meaning `CreateGetOrderLimits` should probably already get projectID and bucket name separately. But yeah, we can leave these for future.